docs/installation/server-config/security.rst

```diff
@@ -0,0 +1,163 @@
+        sasl.mechanism=PLAIN
+
+        # Schema Registry using HTTPS
+        ksql.schema.registry.url=https://schemaregistry:8082
+
+        # Producer Confluent Monitoring Interceptors for Control Center streams monitoring
+        producer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
+        producer.confluent.monitoring.interceptor.ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+        producer.confluent.monitoring.interceptor.ssl.truststore.password=confluent
+        producer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        producer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        producer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        producer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        producer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        producer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+        # Consumer Confluent Monitoring Interceptors for Control Center streams monitoring
+        consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+        consumer.confluent.monitoring.interceptor.ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+        consumer.confluent.monitoring.interceptor.ssl.truststore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        consumer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        consumer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        consumer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        consumer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+    The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+    are signed. For full details, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__.
+
+#.  Start KSQL server with your configuration file specified.
+
+    .. code:: bash
+
+        $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+    .. tip:: After KSQL is started, you can view your settings with the ``SHOW PROPERTIES;`` KSQL command.
+
+#.  If you are using KSQL with HTTPS to Confluent Schema Registry, you must set the ``ksql.schema.registry.url`` to HTTPS
```

Comment:
I'd make this a dedicated "Using KSQL with a secured Confluent Schema Registry" section.